packages/fast-animation/README.md

```diff
@@ -30,7 +33,7 @@ var mySequence = new AnimateSequence([
     new AnimateTo(element3, { x: 20 })
 ]);
 
-// Attach an onFinish callback
+// Attach an onFinish callback.
```

Comment:
nit: this period feels funny to me. Is this the new way we want to do comments? Most comments feel like statements most of the time to me

Reply:
This does feel more like a fragment

Reply:
But it is not a fragment, it has an independent clause with a subject, a verb, and a complete thought.
Subject "onFinish callback" and a verb "Attach".